src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests/CustomConverterTests.NullValueType.cs

```diff
@@ -75,7 +75,7 @@ public class JsonNullableDateTimeOffsetConverter : JsonConverter<DateTimeOffset?
                 }
 
                 string value = reader.GetString();
-                if (value == string.Empty)
+                if (value.Length == 0)
```

Comment:
GetString can return null.  How do you know it isn't in this case?

Reply:
```suggestion
                if (value == string.Empty)
```